internal/plugins/openshift/v1/init.go

```diff
@@ -84,12 +84,42 @@ var imageSubstitutions = map[string][]substitution{
 			regexp.MustCompile(`gcr.io/distroless/static:[^ \n]+`),
 			"registry.access.redhat.com/ubi8/ubi-minimal:" + ubiMinimalVersion,
 		},
+		// Go - for https://access.redhat.com/security/cve/CVE-2023-44487 &&
+		// https://access.redhat.com/security/cve/CVE-2023-39325 .
+		// This will ensure all Go projects have their Dockerfile updated to use
+		// Go 1.20+ as the builder image. This should be removed when the default
+		// scaffolds are updated to be Go 1.20+
+		{
+			regexp.MustCompile(`golang:[^ \n]+`),
+			"golang:1.20",
+		},
```

Comment:
This seems risky. Are we absolutely sure that golang:1.20 will work with projects originally tested with older versions. In theory, this should be okay, but in practice, I don't know.

I was thinking more along the lines of `replace("golang:1.x", rh-go-builder:1.x")` (handwaving over the registry/repo for the right rh go builder image)

Reply:
That's fair. I'll look into switching this around.

Reply:
Maybe I'm looking in the wrong place, but all I'm finding for builder images is `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.15` - will customers have access to this image seeing as it is pulling from a ci registry? is there a different registry or place to find these images than in our openshift/release configs?